Source/Mosa.Compiler.Framework/Stages/BitTrackerStage.cs

```diff
@@ -83,27 +81,19 @@ public Value(uint value)
 			public bool AreUpper32BitsKnown { get { return (BitsKnown & Upper32BitsSet) == Upper32BitsSet; } }
 			public bool AreRangeValuesIndeterminate { get { return !AreRangeValuesDeterminate; } }
 			public ulong BitsKnown { get { return BitsSet | BitsClear; } }
-			public bool Is32Bit { set { IsEvaluated = true; Set32Bit(); } }
-			public bool Is64Bit { set { IsEvaluated = true; } }
-			public bool IsEvaluated { get; private set; }
+			public bool Is32Bit { set { Set32Bit(); } }
```

Comment:
This is risky because it is order dependent, setting this field before others could cause bugs.
Since this is a private sealed class I think it's okay but maybe should be documented.

Reply:
Spot on! I don’t like this either - it’s slightly better than before - but at the next refactoring, I’ll remove the order dependency!